melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/bucinfo/BucInfo.java

```diff
@@ -8,4 +8,12 @@
     private String id;
     private String processDefinitionId;
     private String status;
+
+    public boolean norgeErCaseOwner() {
+        return "PO".equalsIgnoreCase(applicationRoleId);
```

Comment:
Hva er PO her? Kanskje en konstant for litt mer kontekst.

Reply:
PO -> Process Owner (når vi opprettet BUCen). Konstant er nok smart